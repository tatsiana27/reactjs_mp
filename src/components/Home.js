import React, { Component } from 'react';
import { Layout } from './Layout';
import { Header } from './Header';
import { SearchBlock } from './SearchBlock';
import { InfoBlock } from './InfoBlock';
import { MoviesList } from './MoviesList';

export class Home extends Component {
    render() {
        return (
            <Layout>
                <Header>
                    <SearchBlock/>
                </Header>
                <InfoBlock/>
                <MoviesList />
            </Layout>
        )
    }
};