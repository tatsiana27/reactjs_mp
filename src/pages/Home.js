import React, { Component } from 'react';
import { Layout } from './Layout';
import { Header } from '../components/Header';
import { SearchBlock } from '../components/SearchBlock';
import { InfoBlock } from '../components/InfoBlock';
import { MoviesList } from '../components/MoviesList';

export class Home extends Component {

    render() {
        return (
            <Layout>
                <Header>
                    <SearchBlock/>
                </Header>
                <InfoBlock/>
                <MoviesList/>
            </Layout>
        )
    }
};
