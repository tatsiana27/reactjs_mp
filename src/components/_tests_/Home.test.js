import React from 'react';
import { Home } from '../Home';
import { shallow } from 'enzyme';

describe('<Home />', () => {
    test('should render Home Page Component', () => {
        const wrapper = shallow(
            <Home />
        );

        expect(wrapper).toMatchSnapshot();
    });
});