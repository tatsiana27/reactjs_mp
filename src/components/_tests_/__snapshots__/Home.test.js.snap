// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should render Home Page Component 1`] = `
<Layout>
  <Header>
    <SearchBlock />
  </Header>
  <InfoBlock />
  <MoviesList />
</Layout>
`;
